udata_datalake_service/consumer.py

```diff
@@ -0,0 +1,43 @@
+import json
+import logging
+import os
+
+from udata_datalake_service.background_tasks import manage_resource
+from kafka import KafkaConsumer
+
+KAFKA_HOST = os.environ.get("KAFKA_HOST", "localhost")
+KAFKA_PORT = os.environ.get("KAFKA_PORT", "9092")
+KAFKA_API_VERSION = os.environ.get("KAFKA_API_VERSION", "2.5.0")
+TOPICS = os.environ.get("TOPICS", ["dataset"])
```

Comment:
This will change in the future (I think we will use `resource-crawler` topic from decapode actions) but let it that way for now.

Reply:
Outdated